### DISCUSSION TITLE
Feature Request: self-service password reset

### DISCUSSION THREAD

**Discussion by junlincc** [apache/superset#28128] [Ideas]
A few individuals have reached out to request improving existing sign-in flow and to include self-service password reset. 
<img width="1441" alt="Screen Shot 2020-09-23 at 8 45 02 AM" src="https://user-images.githubusercontent.com/67837651/94036378-18f80900-fd79-11ea-986a-6db105bc9425.png">

After users change USERNAME and PASSWORD, if they fail to log in due to an invalid password or username, there is no way to reset. 
If anyone knows any workarounds, please share your solutions. 

(post on behalf)

[a comment by an automated account is not shown]

**Comment by sfirke** [comment]
I see this was marked as completed but as far as I can tell, this feature is missing in Superset 3.1.1.  I reopened this issue and converted it to this discussion as it feels like a request for a missing feature rather than a bug.
